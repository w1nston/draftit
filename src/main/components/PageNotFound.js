import React from 'react';

const PageNotFound = () => <h2>404 - Page Not Found</h2>;

export default PageNotFound;
