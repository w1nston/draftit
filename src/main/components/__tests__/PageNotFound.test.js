import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PageNotFound from '../PageNotFound';

describe('PageNotFound', () => {
  it('renders', () => {
    const component = shallow(<PageNotFound />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
