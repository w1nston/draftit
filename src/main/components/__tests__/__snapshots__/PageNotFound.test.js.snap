// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders 1`] = `
<h2>
  404 - Page Not Found
</h2>
`;
